site/src/Tour.jsx

```diff
@@ -35,6 +36,24 @@ const Steps = [
     placement: "auto",
     offset: 0,
   },
+  {
```

Comment:
Above on line 32, We should change this sentence now that we alter both visibility and styling. 


`content: "These options change style and visibility of the different layers",`